example-1/script.js

```diff
@@ -97,7 +99,7 @@ function enableForm() {
 /**
  * Submit handler
  */
-form.addEventListener("submit", async event => {
+form.addEventListener("submit", async (event) => {
```

Comment:
should we make sure this work in es5 too?  meaning add a comment in the readme? and maybe even in this file saying polyfill this?